doc/source/devguide/index.rst

```diff
@@ -22,8 +22,19 @@ Getting Started
 To get started with development for InstrumentKit, a few additional supporting
 packages must be installed. The core development packages can be found in
 the supporting requirements file named ``dev-requirements.txt``. These will
-allow you to run the tests and check that all your code changes follow our
-linting rules (through `pylint`).
+allow you to run the tests.
+
+This repo also contains a series of static code checks that are managed
+via `pre-commit`. This tool, once setup, will manage running all of these
+checks prior to each commit on your local machine.::
+
+$ pip install pre-commit
+$ pre-commit install
+
+These checks are also run in CI, and must pass in order to generate
+a passing build. It is suggested that you install the git hooks, but
+they can be run manually on all files. See the `pre-commit` homepage
```

Comment:
Would be nice to put a link here for the user: https://pre-commit.com/#3-install-the-git-hook-scripts
The `pre-commit install` does the installation, correct?

Reply:
Yes, `pre-commit install` installs the hooks. The cool thing is that as the configuration file changes and updates, the tool automatically updates all the plugins!